src/app/features/mondomarket/index.js

```
@@ -0,0 +1,243 @@
+      })
+      .addCase(sendOffer.fulfilled, (state, action) => {
+        state.isLoading = false;
+      })
+      .addCase(sendOffer.rejected, (state, action) => {
+        state.isLoading = false;
+        state.error = action?.payload;
+      })
+      .addCase(searchMondoItem.pending, (state) => {
+        state.isSearching = true;
+      })
+      .addCase(searchMondoItem.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)